src/main/ml-modules/root/ds/lux/versionInfo.mjs

```diff
@@ -0,0 +1,37 @@
+import { CODE_VERSION } from '../../lib/appConstants.mjs';
+import { handleRequest } from '../../lib/requestHandleLib.mjs';
+
+handleRequest(function () {
+  return {
+    codeVersion: CODE_VERSION,
+    dataVersion: _getDataConversionDate(),
+    mlVersion: xdmp.version(),
+    databaseName: xdmp.databaseName(xdmp.database()),
+  };
+});
+
+function _getDataConversionDate() {
```

Comment:
Does it matter which document you get?  Here's an alternative that also doesn't throw an exception upon experiencing an issue.

```javascript
function _getDataConversionDate() {
  try {
    return cts
      .doc(fn.subsequence(cts.uris(), 1, 1))
      .xpath('/admin/conversion-date');
  } catch (e) {
    return 'error';
  }
}
```